test/e2e/network/service.go

```diff
@@ -1546,6 +1547,82 @@ var _ = SIGDescribe("Services", func() {
 		jig.ChangeServiceType(svc.Namespace, svc.Name, v1.ServiceTypeClusterIP, createTimeout)
 	})
 
+	// This test creates a load balancer, make sure its health check interval
+	// equals to gceHcCheckIntervalSeconds. Then the interval is manipulated
+	// to be something else, see if the interval will be reconciled.
+	It("should reconcile LB health check interval [Slow]", func() {
```

Comment:
I think this needs to be [serial] no? Otherwise, you might have the healthcheck recreated by another test running at the same time?

Reply:
You're right. This test might be disrupted.